### PR TITLE
docs(applications): explain how to add user ssh key

### PR DESCRIPTION
**Pull request by rimusz**

### CODE REVIEW

**Comment by rimusz** [comment]
replaces #295

File: src/applications/using-buildpacks.md [hunk 1]
```diff
@@ -99,21 +114,6 @@ To use a custom buildpack, set the `BUILDPACK_URL` environment variable.
 
 On your next `git push`, the custom buildpack will be used.
 
-
-## Compile Hooks
```
Comment (bacongobbler) [inline src/applications/using-buildpacks.md]: Looks like some documentation was accidentally removed here.

File: src/applications/using-dockerfiles.md [hunk 2]
```diff
@@ -19,7 +34,6 @@ In order to deploy Dockerfile applications, they must conform to the following r
 * The Dockerfile must use the `EXPOSE` directive to expose exactly one port.
 * That port must be listening for an HTTP connection.
 * The Dockerfile must use the `CMD` directive to define the default process that will run within the container.
-* The Docker image must contain [bash](https://www.gnu.org/software/bash/) to run processes.
```
Comment (bacongobbler) [inline src/applications/using-dockerfiles.md]: same here

**Comment by rimusz** [comment]
used my old edited files, maybe that was the reason

**Comment by rimusz** [comment]
@bacongobbler updated the missing text. thanks for spotting it

**Comment by bacongobbler** [comment]
looks like you'll need to squash the commits, otherwise looks good to me

**Comment by rimusz** [comment]
ah, not again that `squashing` business, did not go well last time. it is just two commits, let's merge it :)

**Pull request merged**